config/environments/development.rb

```diff
@@ -34,4 +34,6 @@
   config.assets.quiet = true
 
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.stripe.secret_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
When I see code repetition I would assume you can place this in general settings like `application.rb` Also, followingyour logic, why not include this into `environments/production.rb`?